website/content/en/preview/getting-started/migrating-from-cas/scripts/step09-deploy.sh

```diff
@@ -0,0 +1,4 @@
+kubectl create namespace karpenter
+kubectl create -f \
+    https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef >}}charts/karpenter/crds/karpenter.sh_provisioners.yaml
```

Comment:
I think you could just use the `$KARPENTER_VERSION` here too. The preview docs will use the latest version rather than main, but that's probably fine for the CRD.